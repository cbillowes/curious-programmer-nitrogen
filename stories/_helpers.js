import React from 'react'
import Layout from '../src/components/layout'
import { SmallSpacer } from '../src/components/spacer'

const Page = ({ className, children }) => {
  return (
    <Layout
      className={className}
    >
      <SmallSpacer />
      {children}
    </Layout>
  )
}

export default Page