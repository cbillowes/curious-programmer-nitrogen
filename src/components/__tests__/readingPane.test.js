import React from "react"
import ReadingPane from "../readingPane"
import { getSnapshot } from "./_helpers"

describe(`Reading pane`, () => {
  it(`should render`, () => {
    const component = (
      <ReadingPane heading="Don't panic.">
        <p>
          One early morning, Arthur's friend, who's actually an alien, informs
          him of Earth's impending doom and whisks him away in a spaceship.
          While Arthur deals with his woes, he also has a great adventure.
        </p>
      </ReadingPane>
    )
    const tree = getSnapshot(component)
    expect(tree).toMatchSnapshot()
  })
})
