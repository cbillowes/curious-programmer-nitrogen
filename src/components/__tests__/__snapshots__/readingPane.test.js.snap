// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reading pane should render 1`] = `
<div>
  <div
    className="spacer small"
  />
  <h1>
    Don't panic.
  </h1>
  <div
    className="reading-pane"
  >
    <p>
      One early morning, Arthur's friend, who's actually an alien, informs him of Earth's impending doom and whisks him away in a spaceship. While Arthur deals with his woes, he also has a great adventure.
    </p>
  </div>
</div>
`;
